packages/protocol/contracts/common/FeeCurrencyWhitelist.sol

```diff
@@ -6,11 +6,19 @@ import "./interfaces/IFeeCurrencyWhitelist.sol";
 
 import "../common/Initializable.sol";
 
+import "../common/interfaces/ICeloVersionedContract.sol";
+
 /**
  * @title Holds a whitelist of the ERC20+ tokens that can be used to pay for gas
  */
-contract FeeCurrencyWhitelist is IFeeCurrencyWhitelist, Ownable, Initializable {
+contract FeeCurrencyWhitelist is
+  IFeeCurrencyWhitelist,
+  Ownable,
+  Initializable,
+  ICeloVersionedContract
+{
   address[] public whitelist;
+  address[] public nonMentoTokenWhitelist;
```

Comment:
We could consider using AddressSet from OpenZeppelin - this would take care of possible duplicities